src/domain/CloseChoices.ts

```diff
@@ -5,8 +5,12 @@ export enum CloseChoices {
 	Hide = 'hide',
 	/** user clicked maybe later button */
 	MaybeLater = 'maybe-later',
+	/** user clicked maybe later button, banner display will be prevented for 7 days */
+	MaybeLater7Days = 'maybe-later',
```

Comment:
You don't need to add new choices here, as they're being mapped for the legacy tracking in https://github.com/wmde/fundraising-banners/blob/main/src/tracking/LegacyEventTracking/mapCloseEvent.ts.

What you can do is have the SoftClose X and the AlreadyDonated maybe later fire `CloseChoices.Close` events, then catch them in that mapper to make them distinguishable.

As they then fire normal close events they'll automatically get the 7 days cookie. You can see that here https://github.com/wmde/fundraising-banners/blob/8e3ae9100b525e26db1b4e304bcf2ffcbb47652b/src/page/PageWPORG.ts#L116 (`preventBannerDisplayForPeriod` adds the 7 day cookie, we should update that name at some point)